.github/workflows/build-tarball.yml

```diff
@@ -4,20 +4,15 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
   push:
-    branches:
-      - master
-      - main
-      - v[0-9]+.x-staging
-      - v[0-9]+.x
+    branches: [master, main, 'v[0-9]+.x-staging', 'v[0-9]+.x']
```

Comment:
I find multiline arrays easier to read (and also to review when elements are added/removed)

Reply:
The line just above is a single line areay, and the two lines have almost the same size. Personally I find that when the inline version is short it's easier to read, otherwise I leave it multiline, like in `coverage-linux.yml`.
But if you prefer multiline arrays, there is no problem, I would put array in multiline.

Reply:
yeah, I put my comment on the wrong line. I don't feel strongly about the `types` and `branches` fields, but I would like at least the ` paths-ignore` to stay multiline